### PR TITLE
Update review article

### PR DESCRIPTION
**Pull request by matks**
Update of article https://build.prestashop.com/news/the-review-process/ as our CI has evolved

### CODE REVIEW

**Comment by matks** [comment]
Someone can review this PR ? It's a small change and it's necessary to make the post accurate regarding our latest changes in CI

File: news/_posts/2019-09-10-the-review-process.md [hunk 1]
```diff
@@ -92,3 +92,6 @@ As you can see, the process between the submission of Pull Request and its merge
 A downside of this long journey and the level of commitment it requires from both the reviewers and the authors is that some Pull Requests require a very long time to be reviewed. Also, we are actually not a big team. As of September of 2019, there are only 6 Core maintainers available to review more than 400 Pull Requests... this is why PrestaShop is currently hiring!
 
 We also are working on other ideas to bring more people in the maintainer team and to be able to review/merge everything, so we hope to get better in the following months.
+
+[1]: At the time of publication of this article, there was only two: PrettyCI and TravisCI.
+[2]: This used to be done by a 3rd party service known as PrettyCI but it has [shut down](https://prettyci.com/goodbye).
```
Comment (ttoine) [inline news/_posts/2019-09-10-the-review-process.md]: Adding a comment to tell that the article has been updated is a good practice.

```suggestion
[2]: This used to be done by a 3rd party service known as PrettyCI but it has [shut down](https://prettyci.com/goodbye).

Note: the original article has been edited on Friday 3rd of April 2020 to remove PrettyCI and replace it in the content by new solutions.
```

**Comment by matks** [comment]
I updated the PR to use a glossary as bottom page references dont work (see https://github.com/PrestaShop/prestashop.github.io/issues/670)

**Comment by ttoine** [comment]
ok for me, then

**Comment by ttoine** [comment]
thanks for the improvement.

**Pull request merged**